ultradns/src/test/java/denominator/ultradns/UltraDNSLiveTest.java

```diff
@@ -0,0 +1,64 @@
+package denominator.ultradns;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+import denominator.DNSApiManagerFactory;
+import denominator.Live;
+
+import static feign.Util.emptyToNull;
+import static java.lang.System.getProperty;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    UltraDNSLiveTest.CheckConnectionLiveTest.class,
+    UltraDNSLiveTest.ReadOnlyLiveTest.class,
+    UltraDNSLiveTest.WriteCommandsLiveTest.class,
+    UltraDNSLiveTest.RoundRobinWriteCommandsLiveTest.class,
+    UltraDNSLiveTest.GeoReadOnlyLiveTest.class,
+    UltraDNSLiveTest.GeoWriteCommandsLiveTest.class
+})
+public class UltraDNSLiveTest {
```

Comment:
@jonbodner @jdamick Hopefully, centralizing this into one file keeps things more sane. There's mild refactoring opportunity, but I decided to stop as otherwise I'll get no more work done for the next month :) Anyway, feel free to start on the new rest provider in master, as the base test framework is at or near where I feel it should be in order to facilitate easier work in the future.